cachito/paths.py

```diff
@@ -0,0 +1,38 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import logging
+import shutil
+
+from pathlib import Path
+
+log = logging.getLogger(__name__)
+
+
+class RequestBundleDir(type(Path())):
```

Comment:
Could you add a comment explaining that this will use `PosixPath` on Linux and `WindowsPath` on Windows as the base class?

If someone doesn't know this, it seems quite strange.